src/components/Clock.js

```diff
@@ -0,0 +1,94 @@
+/* eslint-disable react/button-has-type */
+/* eslint-disable consistent-return */
+
+import React from 'react';
+import './Clock.scss';
+
+export class Clock extends React.Component {
+  state = {
+    date: new Date(),
+    name: '$',
+    visibility: false,
+    previousName: '1',
+  };
+
+  constructor() {
+    super();
+    this.handleClickClock = this.handleClickClock.bind(this);
+    this.handleClickName = this.handleClickName.bind(this);
+  }
+
+  componentDidMount() {
+    setInterval(() => {
+      this.setState({ date: new Date() });
+      // eslint-disable-next-line
+    console.log(this.state.date.toLocaleTimeString());
+    }, 1000);
+  }
+
+  handleClickClock() {
+    this.setState(state => ({
+      visibility: !state.visibility,
+    }));
+  }
+
+  handleClickName() {
+    this.setState(state => ({
+      previousName: state.name,
+      name: Math.floor(Math.random() * (Math.floor(100) - Math.ceil(0)))
+       + Math.ceil(0),
```

Comment:
```suggestion
       + 0,
```
<img width="187" alt="image" src="https://user-images.githubusercontent.com/38065883/93781890-89e0d900-fc32-11ea-8208-14c42d50c262.png">

Reply:
do you need this part? it will work same way without it